sdk/src/main/scala/com/stratio/sparkta/sdk/ValidatingPropertyMap.scala

```diff
@@ -46,8 +46,8 @@ class ValidatingPropertyMap[K, V](val m: Map[K, V]) {
 
   def getString(key: K, default: Option[String]): Option[String] = {
```

Comment:
It is just used in ElasticSearchOutput right? In 
`override val defaultIndexMapping = properties.getString("indexMapping", Some(DEFAULT_INDEX_TYPE))`

defaultIndexMapping always is going to have a value so there is no point to treat it as an Option

Here we could use
`override val defaultIndexMapping = properties.getString("indexMapping", DEFAULT_INDEX_TYPE)`